test/scripts/mysqlx/behavioral_validation.py

```
@@ -104,18 +94,47 @@ def steady_traffic_thread(args, stop_event, results: List[dict], idx: int):
+    try:
+        adm = mysql.connector.connect(
+            host=args.admin_host, port=args.admin_port,
+            user=args.admin_user, password=args.admin_pass,
+            ssl_disabled=True,
+        )
+        cur = adm.cursor()
+        try:
+            cur.execute("PROXYSQL SHUTDOWN")
+            print("Admin shutdown command accepted.")
+            return True
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)